process-dataset/steps/write-per-cell-jsons.js

```diff
@@ -48,6 +56,11 @@ const formatAndWritePerCellJsons = async (readFolder, outFolder, featureDataFile
                     i: fileInfo.CellId,
                 }
             }
+            /* end of feature json loop */
+
+            map(counts, (value, key) => {
+                firebaseHandler.updateFeatureCount(defaultGroupBy, key, value)
+            })
```

Comment:
what does updateFeatureCount do?

Reply:
this saves the counts for each "groupBy" category so we can have them on the checkbox bar on the left. In the future it might make sense to just do this for every feature. it's stored on the feature def, which maybe isn't great since it's not really a definition. Before it was stored on the cell line